sdks/python/apache_beam/transforms/ptransform.py

```diff
@@ -92,13 +93,36 @@ class and wrapper class that allows lambda functions to be used as
     'label_from_callable',
     ]
 
+
 T = TypeVar('T')
 PTransformT = TypeVar('PTransformT', bound='PTransform')
 ConstructorFn = Callable[
     [Optional[Any], 'PipelineContext'],
     Any]
 
 
+KNOWN_COMPOSITES = frozenset([
+    common_urns.primitives.GROUP_BY_KEY.urn,
+    common_urns.composites.COMBINE_PER_KEY.urn,
+    common_urns.primitives.PAR_DO.urn,  # After SDF expansion.
+])
+
+
+COMBINE_URNS = frozenset([
+    common_urns.composites.COMBINE_PER_KEY.urn,
+])
+
+
+PAR_DO_URNS = frozenset([
```

Comment:
This feels a bit out of place here, I'd move it back to where it was.

Reply:
Moved back to original location.